### ISSUE TITLE
nicer formating of grid-template-areas

### ISSUE THREAD

**Issue by jogibear9988** [adobe/css-tools#283]
### Expected Behaviour
template areas should be stringified like this:

    grid-template-areas: "menu head head head"
                         "menu main main main"
                         "menu main main main"
                         "menu footer footer footer";

### Actual Behaviour

    grid-template-areas: "menu head head head"
    "menu main main main"
    "menu main main main"
    "menu footer footer footer";

**Comment by holblin** [comment]
Yes it could be good, though I'm not sure of the priority for this work.
I mainly try to remove security issue and maintain the package but I have no time to improve it.
Feel free to make a PR with that changes and add associated test though, I would be happy to review and merge it.

**Comment by holblin** [comment]
This will be released in the version 4.4.3
